### DISCUSSION TITLE
No equivalent command for Daemonization

### DISCUSSION THREAD

**Discussion by partha117** [celery/django-celery-beat#471] [Ideas]
Hi, I want to reproduce the exact behavior in the beat daemon. But can not found any command for daemonization.
   Normally I write `celery -A some_proj worker -l info -B -c 2 --max-tasks-per-child 1 -Q some_periodic -n vts_app`
I have specified the queue name in the beat. But the problem is in daemon I can not specify the queue. It only takes --schedule as an argument.
 So what will be the equivalent daemon version of the command?

**Comment by auvipy** [comment]
were you able to do it?

**Comment by partha117** [comment]
Nope
